webapi/helpers.go

```diff
@@ -88,3 +99,64 @@ func isValidTicket(tx *wire.MsgTx) error {
 
 	return nil
 }
+
+// validateTicketHash ensures the provided ticket hash is a valid ticket hash.
+// A ticket hash should be 64 chars (MaxHashStringSize) and should parse
+// into a chainhash.Hash without error.
```

Comment:
If this doesn't exceed the max 80 chars per line limit, this is better formatted:
```suggestion
// validateTicketHash ensures the provided ticket hash is a valid ticket hash.
// A ticket hash should be 64 chars (MaxHashStringSize) and should parse into
// a chainhash.Hash without error.
```